client/termdb/Vocab.js

```diff
@@ -173,17 +173,32 @@ export class Vocab {
 	async addGroup(obj) {
 		// save one custom term
 		// obj = { name:str, term:{} }
+		this.state.groups.push(obj)
+
 		await this.app.dispatch({
-			type: 'add_group',
-			obj
+			type: 'app_refresh',
+			state: { groups: this.state.groups }
 		})
 	}
+
 	async deleteGroup(name) {
-		// delete by name
-		await this.app.dispatch({
-			type: 'delete_group',
-			name
-		})
+		const i = this.state.groups.findIndex(i => i.name == name)
+		if (i != -1) {
+			const group = this.state.groups[i]
+			this.state.groups.splice(i, 1)
+			await this.app.dispatch({
+				type: 'app_refresh',
+				state: { groups: this.state.groups }
+			})
+			const config = this.state.plots.find(p => p.id === group.plotId)
```

Comment:
thanks looks ok to me. @siosonel what do you think about dispatching twice in a row

Reply:
I don't think two dispatches are needed. 
- When changing a tw group for a chart's config, then dispatch only `plot_edit`
- When renaming or editing a group, that could affect other charts, then should have a `group_edit` action in the store (would require updating `reactsTo()` of each chart class), OR use app_refresh (wouldn't require updating reactTo()).

Reply:
Solved now. Edgar helped me to review the reacting logic.